packages/tasks/src/tasks/video-text-to-text/data.ts

```diff
@@ -0,0 +1,59 @@
+import type { TaskDataCustom } from "..";
+
+const taskData: TaskDataCustom = {
+	datasets: [
+		{
+			description: "Multiple-choice questions and answers about videos.",
+			id: "lmms-lab/Video-MME",
+		},
+        {
+			description: "A dataset of instructions and question-answer pairs about videos.",
+			id: "lmms-lab/VideoChatGPT",
+		},
+		
+	],
+	demo: {
+		inputs: [
+			{
+				filename: "video-text-to-text-input.gif",
+				type: "video",
```

Comment:
This type does not exist [here](https://github.com/huggingface/huggingface.js/blob/4959181d3c43ad5deb1c2a3a05db94abfe358dbe/packages/tasks/src/tasks/index.ts#L254). I don't know if it should be added in https://github.com/huggingface-internal/moon-landing/blob/main/server/views/components/TaskSchemaEntry/TaskSchemaEntry.svelte and, in that case, if it needs a new widget type.

As an alternative, we could use `"img"`, but not sure if the `gif` will be rendered. Or prepare a static image that shows a few frames.

What do you think @osanseviero?

Reply:
ah yes sorry it should be image